src/modules/land_detector/LandDetector.cpp

```diff
@@ -164,6 +156,11 @@ void LandDetector::_check_params(const bool force)
 	}
 }
 
+void LandDetector::_update_params()
```

Comment:
How about just declaring a default parameter for `_update_params()`?
I'll add the suggestion at the declation line.

Reply:
I missread it now I see that it's already overloaded... I think others might struggle as well since it's quite confusing.

Reply:
Thanks @MaEtUgR , I agree and like your suggestion.  I've pushed up a commit to address.  I need to re-flight test and will get that done in a few hours then post logs.

Reply:
Sounds good 👍